test/eventsource.h

```diff
@@ -24,6 +24,12 @@
 #include <rapidjson/document.h>
 #include <vector>
 
+#if defined(__FreeBSD__)
+#include <unistd.h>
+typedef u_long ulong;
```

Comment:
This is probably also an issue on Linux under musl. Prefer unconditional as in https://github.com/ohwgiles/laminar/blob/master/src/http.h#L29